components/automate-chef-io/layouts/404.html

```diff
@@ -0,0 +1,33 @@
+<!doctype html>
+<html lang="en">
+{{ partial "automate/head" . }}
+
+<body>
+  <div class="off-canvas-wrapper has-sidebar">
+    <div class="off-canvas-content" data-off-canvas-content>
+      <div id="menu" class="cell small-12" data-sticky-container>
+        {{ partial "automate/header" . }}
+      </div>
+      <div class="cell small-12" style="z-index: 10;">
+        {{ partial "mini-hero" . }}
+      </div>
+      <div id="main-content" class="cell small-12" data-sticky-container>
+        <div id="main-content-col" class="col-content">
+          <h1>404</h1>
+          <div class="prose">
+            <p>The page you were looking for is not here. If you think it's our fault,
```

Comment:
maybe...

`We did something wrong! This page doesn't exist, <a href="https://docs.chef.io/feedback.html">reach out to Chef</a> if you need help.